packages/api/src/controllers/taskController.js

```diff
@@ -658,26 +671,39 @@ const TaskNotificationTypes = {
   TASK_ABANDONED: 'TASK_ABANDONED',
   TASK_REASSIGNED: 'TASK_REASSIGNED',
   TASK_COMPLETED_BY_OTHER_USER: 'TASK_COMPLETED_BY_OTHER_USER',
+  TASK_UNASSIGNED: 'TASK_UNASSIGNED',
 };
 
 const TaskNotificationUserTypes = {
   TASK_ASSIGNED: 'assignee',
   TASK_ABANDONED: 'abandoner',
   TASK_REASSIGNED: 'assigner',
   TASK_COMPLETED_BY_OTHER_USER: 'assigner',
+  TASK_UNASSIGNED: 'editor',
 };
 
+/**
+ * Sends a notification to the specified receivers about a task
+ * @param {Array<String>} receiverIds
+ * @param {String} senderId
+ * @param {String} taskId
+ * @param {String} notifyTranslationKey
+ * @param {String} taskTranslationKey
+ * @param {String} farmId
+ * @return {Promise<void>}
+ */
+
 async function sendTaskNotification(
-  receiverId,
+  receiverIds,
   senderId,
   taskId,
   notifyTranslationKey,
   taskTranslationKey,
   farmId,
 ) {
-  if (!receiverId) return;
+  if (receiverIds.includes(null) || receiverIds.includes(undefined)) return;
```

Comment:
@smattingly this was originally implemented to prevent a notification from being sent to a non existent user when a task was assigned/re-assigned to or from an Unassigned status. I'd be open to taking a different approach to dealing with that case, but my current approach was to just preserve the existing logic.

Reply:
For now, I will filter `receieverIds` to remove null and undefined values, and if it has a length greater than 0, I will continue, otherwise I will return early